app/helpers/description_change_request_helper.rb

```diff
@@ -0,0 +1,21 @@
+module DescriptionChangeRequestHelper
+  def display_request_date_state(description_change_request)
+    if description_change_request.state == "closed"
+      "grey"
+    elsif description_change_request.days_until_response_due.positive?
+      "green"
+    else
+      "red"
+    end
+  end
+
+  def display_request_status(description_change_request)
+    if description_change_request.state == "open"
+      "grey"
+    elsif description_change_request.approved == false
```

Comment:
I was going to suggest using `!description_change_request.approved?` but I don't know if using an `if` negatively is better than what you have?

Reply:
because the approved can be nil if it is in pending, in which case it is still !approved, so that's why this uses the comparison to false instead